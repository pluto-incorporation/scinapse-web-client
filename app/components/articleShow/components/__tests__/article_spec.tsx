jest.unmock("../article");

import * as React from "react";
import { shallow, ShallowWrapper } from "enzyme";
import Article, { IArticleProps } from "../article";

describe("<Article /> component", () => {
  let articleWrapper: ShallowWrapper<IArticleProps>;
  const mockLink = "https://pluto.network";

  beforeEach(() => {
    articleWrapper = shallow(<Article link={mockLink} />);
  });

  it("should match snapshot", () => {
    expect(articleWrapper.html()).toMatchSnapshot();
  });
});
