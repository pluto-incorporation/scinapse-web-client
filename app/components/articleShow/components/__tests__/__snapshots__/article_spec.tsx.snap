// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Article /> component should match snapshot 1`] = `"<div class=\\"articleWrapper\\"><div class=\\"title\\">Article</div><a target=\\"_blank\\" href=\\"https://pluto.network\\" class=\\"articleButton\\">Go to read the article</a></div>"`;
